packages/@lwc/integration-karma/test/component/LightningElement.attachInternals/elementInternals/sanity/index.spec.js

```diff
@@ -21,40 +21,6 @@ if (process.env.NATIVE_SHADOW && typeof ElementInternals !== 'undefined') {
             expect(elm.internals.shadowRoot).toBe(elm.shadowRoot);
         });
 
-        it('only allow listed properties accessible', () => {
-            if (process.env.NODE_ENV !== 'production') {
-                // Not allowed setter
-                // Chrome/Firefox/Safari have different messages
-                expect(() =>
-                    expect(() => (elm.internals.foo = 'bar')).toLogWarningDev(
-                        /Only properties defined in the ElementInternals HTML spec are available./
-                    )
-                ).toThrow();
-                // Not allowed getters
-                expect(() => elm.internals.foo).toLogWarningDev(
-                    /Only properties defined in the ElementInternals HTML spec are available./
-                );
-                // Allowed getter
-                expect(() => elm.internals.ariaAtomic).not.toThrow();
-            } else {
-                // Not allowed setter
-                // Chrome/Firefox/Safari have different messages
-                expect(() => (elm.internals.foo = 'bar')).toThrow();
-                // Not allowed getters
-                expect(elm.internals.foo).toBeUndefined();
-                // Allowed getter
-                expect(() => elm.internals.ariaAtomic).not.toThrow();
-            }
-        });
-
-        it('should allow invocations from Object.prototype and symbols', () => {
-            // Inherited from Object.prototype
-            expect(elm.internals.toString()).toContain('ElementInternals');
-            // Calls out to Symbol.toStringTag, will return a false value but reaches into
-            // Symbol.toStringTag to verify
-            expect(() => hasOwnProperty.call(elm.internals, 'ariaAtomic')).not.toThrow();
-        });
-
```

Comment:
These tests should still pass, right? They are just kind of useless now that we don't have a proxy anymore?

Reply:
Yup! They still pass, I just removed them because we had an allow-list of fields available on the proxy object.

Now that we're going with the native `ElementInternal` object, I don't think we need to check which fields are on it anymore.